src/NBitcoin/IBitcoinSerializable.cs

```diff
@@ -42,20 +42,28 @@ public static void ReadWrite(this IBitcoinSerializable serializable, byte[] byte
             ReadWrite(serializable, new MemoryStream(bytes), false, consensusFactory, version);
         }
 
+        private static BitcoinStream CreateBitcoinStream()
+        {
+            return new BitcoinStream(Stream.Null, true)
+            {
+                ConsensusFactory = new DefaultConsensusFactory(),
```

Comment:
Possibly I would set that in the constructor

Reply:
ah, just raised an issue with that problem.

Reply:
@dangershony can we look at maybe doing that across the board after this fix has gone in?

Reply:
I think he just means that yo udon't need to have a new static method to instantiate the BitcoinStream object. Just bring the `new BitcoinStream(...` inline to from where it's called.

Reply:
ah no he's saying that you should set up the `DefaultConsensusFactory` in the ctor.
What I'm saying is that you don't need an extra method :-)